db/dao/sector_dao.py

```diff
@@ -25,3 +70,32 @@ def delete_sector(sector, commit=True):
         DB().session.delete(sector)
         if commit:
             DB().session.commit()
+
+    @staticmethod
+    def get_sectors_linked_to_file(file):
```

Comment:
We do not need this method. All sectors are accessible using `sectors` property of every file.

Reply:
still looked cool though :')